tito.props.5.asciidoc

```diff
@@ -80,6 +80,10 @@ This option is used control the text of git commit message that is used when
 new tag is generated. You can use "%(name)s", "%(release_type)s" and
 "%(version)s" placeholders.
 
+tag_format::
+This option controls the format used in UserSpecifiedTagger. If not specified 
+default format would be '{component}-{version}-{release}'. It won't affect 
+other taggers.
```

Comment:
Actually looking at this I'm now wondering if this should be part of the base tagger, not a separate implementation. Would this slip into the base VersionTagger, trigger the behavior if the user specified a tag format?

If not, we should move this out of the more global "buildconfig" section and into a "format_tagger" section.

However curious what you think about bringing this into the core tagger itself.